scripts/cnode-helper-scripts/cntools.sh

```diff
@@ -1255,10 +1223,64 @@ case $OPERATION in
       say "${RED}ERROR${NC}: invalid URL format or more than 64 chars in length"
       waitForInput && continue
     fi
-    
-    say "\n${ORANGE}Please make sure you host your metadata JSON file (with contents as below) at ${meta_json_url} :${NC}\n"
-    say "{\n  \"name\": \"${meta_name}\",\n  \"ticker\": \"${meta_ticker}\",\n  \"description\": \"${meta_description}\",\n  \"homepage\": \"${meta_homepage}\"\n}" | tee "${pool_meta_file}"
-    
+
+    metadata_done=false
+    if wget -q  "$meta_json_url" -P "$pool_meta_save_location"; then
+      say "\nMetadata exists at URL.  Use existing data?\n"
+      case $(select_opt "[y] Yes" "[n] No") in
+        0) filename=$(basename "$meta_json_url")
```

Comment:
Can we leave the name saved within cntools to be poolmeta.json, helps with the consistency as well as troubleshooting support queries. In case of which ${pool_meta_file} is already set to the right value.

Reply:
thats fine.  It would also avoid any accidental overwrites of pool files if the user does something dumb like link to a file called `cold.skey` for example.  I can push up a change to do this today.